docs/examples/Transformer Examples.ipynb

```diff
@@ -386,7 +386,7 @@
     "three_winding_transformer[\"winding_2\"] = [1]\n",
     "three_winding_transformer[\"winding_3\"] = [1]\n",
     "three_winding_transformer[\"clock_12\"] = [5]\n",
-    "three_winding_transformer[\"clock_13\"] = [-7]  # supports periodic input\n",
+    "three_winding_transformer[\"clock_13\"] = [5]  # supports periodic input\n",
```

Comment:
This is a recently added feature (#987 and #1011), so the notebook itself is correct.

However, you are completely right in that it should not throw the validation error. You have found a bug in our data validator.

Would you like to fix that or would you prefer us to do so?

c.c. @Jerry-Jinfeng-Guo

Reply:
I’d prefer someone with expertise in this software and a good understanding of these tests to handle it for accuracy.